docs/content/preview/explore/ysql-language-features/going-beyond-sql/tablespaces.md

```diff
@@ -211,13 +211,29 @@ yugabyte=# SELECT * FROM multi_region_table;
 Time: 337.154 ms
 ```
 
-{{< tip title="Note" >}}
+## Leader preference
 
-The location of the leader can also play a role in the preceding latency, and the numbers can differ
-based on how far the leader is from the client node. However, controlling leader affinity
-is not supported via tablespaces yet. This feature is tracked [here](https://github.com/yugabyte/yugabyte-db/issues/8100).
+Leader preference helps optimize workloads that require distribution of data over multiple zones for zone-level fault tolerance, but which have clients only in a subset of those zones. It overrides the default behavior of spreading the tablet leaders across all placement zones of the tablespace, and instead places them closer to the clients.
 
-{{< /tip >}}
+The leaders handle all [reads](../../../../architecture/core-functions/read-path/) and [writes](../../../../architecture/core-functions/write-path/), which reduces the number of network hops, which in turn reduces latency for increased performance. Leader preference allows you to specify the zones in which to place the leaders when the system is stable, and fallback zones when an outage or maintenance occurs in the preferred zones.
+
+In the following example our tablespace is setup to have replicas in us-east-1, us-east-2 and us-west-1. This enables it to survive the loss of an entire Region. The clients are located in us-east-1. By default we would have a third of the leaders in us-west-1, which has a latency of 62ms from our clients.
```

Comment:
Maybe we should mention that it is reads that will be fast, not writes

Reply:
Both reads and writes get faster by the same amount as both have the same extra network hop from client to leader zone.

Reply:
@hari90 How is that? If you have cross-region replicas, we still have to perform cross-region writes whether your leader is local or not (I am of course, assuming the case where majority of replicas are not local). However since we read only from the leader, reads become fast if the leader is local.